resources/views/projects/create.blade.php

```diff
@@ -0,0 +1,252 @@
+						{{$course['name']}}</option>
+					@endforeach
+				@endif
+			</select>
+		</div>
+		<!-- Category -->
+		<div class="field">
+			<label for="category">Category</label>
+			<select name="category" class="ui search dropdown">
+				<option value="">Finance</option>
+				@if(isset($categories))
+					@foreach($categories -> all() as $category)
+						<option value={{$category['id']}}>{{$category['name']}}</option>
+					@endforeach
+				@endif
+			</select>
+		</div>
+		<!-- Skillset -->
+		<div class="field">
+			<label for="skillsets">Skillsets</label>
+			<select name="skillsets" multiple="" class="ui fluid dropdown">
+				<option value="">Java, HTML</option>
+				@if(isset($skillsets))
+					@foreach($skillsets -> all() as $skillset)
+						<option value={{$skillset['id']}}>
+						{{$skillset['name']}}</option>
+					@endforeach
+				@endif
+			</select>
+		</div>
+		<!-- Milestone -->
+		<div class="field">
+			<label for="milestone">Public Milestone</label>
+			<input type="text" name="milestone" placeholder="Design">
+		</div>
+		<!-- Short Description -->
+		<div class="field">
+			<label for="shortDescription">Brief Description</label>
+			<textarea name="shortDescription" rows="2" placeholder="The project is a web page for personal financial organization"></textarea>
+		</div>
+		<!-- Long Description -->
+		<div class="field">
+			<label for="longDescription">Detailed Description</label>
+			<textarea name="longDescription" placeholder="The project consists of a single page application where users can sign up or login. It includes..."></textarea>
+		</div>
+		<!-- Video Pitch -->
+		<div class="field">
+			<label for="videoPitch">Pitch Video</label>
+			<input type="text" name="videoPitch" placeholder="https://youtube.com/watch?v=238028302">
+		</div>
+		<!-- Register Button -->
+		<button id="registerButton" type="submit" class="ui primary submit button">Register Project</button>
+		<!-- Error Message -->
+		<div class="ui error message"></div>
+	</form>
+</div>
+@section('scripts')
+<script>
+	/* Upload new image */
+	$(".imgUploader").click(function(event) {
+		event.preventDefault();
+		$("#imgInput").click();
+	});
+	function validateImage(file) {
+		const maxImageSize=1000000; // 1MB
+		if((file.type == "image/png" || file.type == "image/jpg") && file.size <= maxImageSize) {
+			return true;
+		}
+		else {
+			return false;
+		}
+	}
+	function updateImage(imageInput) {
+		let reader = new FileReader();
+		let file = imageInput.files[0];
+		let preview = $("#projectImagePreview");
+		reader.addEventListener("load", function() {
+			if(validateImage(file)) {
+				$("#projectImagePreview").attr("src", reader.result);
+			}
+		});
+		if(file) {
+			reader.readAsDataURL(file);
+		}
+	}
+	/* Register button validation */
+	$('.ui.dropdown').dropdown();
+	$('.ui.form').form({
+		fields: {
+			projectImage: {
+				identifier:'projectImage',
+				rules:[{
+						type:'empty',
+						prompt:'Please enter a project image'
+					}
```

Comment:
En estos rules, empiezas los arrays juntandolos con el primer objeto `[{`, pero luego cuando los cierras pones el closing brace `}` y el closing square bracket `]` en dos l&iacute;neas diferentes. De preferencia hay que ser consistentes. Ambos son v&aacute;lidos pero agarrate una and stick to it como convention please 😄